Training/view/frontend/layout/customer_account_index.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Codifi_Training
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+-->
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" content="1column">
```

Comment:
1. "content" param doesn't exist in `urn:magento:framework:View/Layout/etc/page_configuration.xsd`
2. `xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd"` is missed
Please, check how to extend customer_account_index in module Magento_Sales or Magento_Review